src/ray/common/ray_object.cc

```diff
@@ -19,4 +19,13 @@ bool RayObject::IsException() {
   return false;
 }
 
+bool RayObject::IsInPlasmaError() {
```

Comment:
IsPlasmaObject() seems more accurate

Reply:
I think that's somewhat misleading, this is more precise.